controlExtensionProject/src/test/java/CSSProject2/CSS2.java

```diff
@@ -0,0 +1,92 @@
+package CSSProject2;
+import framework.TestBase;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class CSS2 extends TestBase{
+	private ChromeDriver driver;
+	CSS2Page css2 = new CSS2Page(driver);
+
+	@Test
+	public void textbox() {
+		String expectedtextbox = "Full Name";
+		css2.gotoTextbox();
+		String actualtextbox = driver.findElement(By.cssSelector("input#userName")).getAttribute("placeholder");
```

Comment:
You should not find elements at the testcase layer. The Test layer responsibility to define the test data, steps, and assertions.